Remove opencv-python-headless as we'll use the EB OpenCV instead
Patch by Simon Branford (University of Birmingham)
--- setup.py.orig	2021-08-11 11:44:31.239366000 +0100
+++ setup.py	2021-08-11 11:44:40.662227000 +0100
@@ -3,8 +3,7 @@
 
 install_deps = ['numpy', 'scipy', 'natsort',
                 'tifffile', 'tqdm', 'numba', 
-                'torch>=1.6',
-                'opencv-python-headless']
+                'torch>=1.6']
 
 try:
     import torch
